CI/scripts/test.sh

```diff
@@ -424,7 +424,7 @@ function test-run_example-simple_app() {
 
     # Rebuild shared library that pytest needs
     pushd ../src > /dev/null 2>&1
-    make -f certifier.mak --always-make -j2 sharedlib
+    NO_ENABLE_SEV=1 make -f certifier.mak --always-make -j${NumMakeThreads} sharedlib
```

Comment:
The reason for this has been explained in [this note](https://github.com/vmware-research/certifier-framework-for-confidential-computing/issues/215#issuecomment-1783620655) added to the comments in the issue #215